statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %s %d\n", bucket, strconv.FormatFloat(currentValue, 'f', -1, 64), now)
 			lastGaugeValue[bucket] = currentValue
 			gauges[bucket] = math.MaxUint64
```

Comment:
Changes look good. This is more of a nitpick for consistency, would you mind changing this to `MaxFloat64`? @ploxiln also had a good suggestion in #71 for refactoring `gauges` + `lastGaugeValue`.